scenario_player/runner.py

```diff
@@ -89,17 +89,17 @@ def is_udc_enabled(udc_settings: UDCSettingsConfig):
     return udc_settings.enable and should_deposit_ud_token
 
 
-def wait_for_nodes_to_be_ready(node_runners, session):
+def wait_for_nodes_to_be_ready(node_runners: List[NodeRunner], session: Session):
     with gevent.Timeout(MAX_RAIDEN_STARTUP_TIME):
         for node_runner in node_runners:
-            url = f"http://{node_runner.base_url}/api/v1/address"
+            url = f"http://{node_runner.base_url}/api/v1/status"
             while True:
                 try:
-                    if session.get(url).ok:
+                    if session.get(url).json()["status"] == "ready":
                         break
-                except requests.exceptions.RequestException:
+                except (requests.exceptions.RequestException, ValueError, KeyError):
                     pass
-                gevent.sleep(0.5)
+                gevent.sleep(1.0)
```

Comment:
This makes me want to have a long-polling endpoint for the status, which delays returning the status when it is not ready. But for now, that would be over-engineering.